src/index.js

```diff
@@ -94,6 +96,10 @@ function App() {
             path="/dev"
             component={() => <HomePage LoadingComponent={LoadingComponent} />}
           />
+          {/* The above route exists for developmental purposes, but the "/" path will be for the home page ("/landing") in the deployed version */}
+          <SecureRoute path="/admin-add-course" component={AdminAddCourses} />
+          <SecureRoute path="/admin-courses" component={AdminCourses} />
+          <SecureRoute path="/news-feed" component={NewsFeed} />
```

Comment:
doing good, but you might what to define that by Importing that in... just saying.